### DISCUSSION TITLE
Github Codespaces

### DISCUSSION THREAD

**Discussion by tarundatta07** [community/community#64750] [Codespaces] [answered]
How can I work with Github Codespaces?

**Comment by arafajs** [comment]
1. Go to your GitHub repository and click on the "Code" button.
2. Select "Codespaces".
3. If you don't have a Codespaces starter plan, you'll be prompted to create one.
4. Choose a programming language and select a starter code.
5. Click on the "Create Codespace" button.
6. Your Codespace will be created and launched in a new tab in your browser.
7. You can start coding right away!

**Comment by iqbal-web** [comment, marked as answer]
### Codespaces
Use the REST API to manage GitHub Codespaces.

https://docs.github.com/en/rest/codespaces/codespaces?apiVersion=2022-11-28
